contrib/sssd.spec.in

```diff
@@ -139,6 +139,8 @@ Recommends: libsss_sudo = %{version}-%{release}
 Recommends: libsss_autofs%{?_isa} = %{version}-%{release}
 Recommends: sssd-nfs-idmap = %{version}-%{release}
 Requires: libsss_idmap = %{version}-%{release}
+Requires: libsss_certmap = %{version}-%{release}
+Requires(pre): shadow-utils
```

Comment:
(I know this is from RHEL8 spec-file but) do you know the reason for this?
This is probably for the part that creates sssd user?
In this case wouldn't it make sense to wrap this req into "%if 0%{?rhel}"?

Reply:
Yes, it is because we call useradd and groupadd on rhel. Condition added.